stable/gcloud-sqlproxy/templates/pdb.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.podDisruptionBudget -}}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  labels:
+    app: {{ template "gcloud-sqlproxy.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "gcloud-sqlproxy.fullname" . }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "gcloud-sqlproxy.name" . }}
```

Comment:
The selector needs the `release` label as well.

Reply:
Just to be clear, do you want me to change the `Deployment` too? the current pod spec doesn't include the `release` label afaict:

```
  template:
    metadata:
      labels:
        app: {{ template "gcloud-sqlproxy.fullname" . }}
    spec:
```